instrumentation/opentelemetry-instrumentation-celery/src/opentelemetry/instrumentation/celery/__init__.py

```diff
@@ -271,6 +279,18 @@ def _trace_failure(*args, **kwargs):
             return
 
         if ex is not None:
+            # Unwrap the actual exception wrapped by billiard's
```

Comment:
We need some testing for these changes.

Reply:
Is it not tested implicitly [here](https://github.com/open-telemetry/opentelemetry-python-contrib/pull/1863/files#diff-9d6bf208153ddc0d6bd1c4044cd9244a5282a3b05709e991809f5506ce3a4ae6L282-R282)? The exception is now of type `Exception` (the type that was raised) instead of `ExceptionInfo` (the Billiard wrapper)

Reply:
Added an unit test for a failing task.